### ISSUE TITLE
Installation issues

### ISSUE THREAD

**Issue by wdkflag** [rsxdalv/tts-generation-webui#435]
Installation issues
[output.log](https://github.com/user-attachments/files/18276105/output.log)

**Comment by rsxdalv** [comment]
Thanks for the log:

```
'EM' 不不是是内内部部或或外外部部命命令令，，也也不不是是可可运运行行的的程程序序
```
This error happens when windows for some reason does not read the installer scripts correctly.
My only clues are Windows 11 and some encoding issues being the cause. (Perhaps it misinterprets newlines and causes the issue that way.

Here's an example discussion about it:
https://github.com/zadjii/keep/issues/1

A potential issue is also git CRLF vs LF settings.

Are you using WSL or git with special settings per chance?

**Comment by wdkflag** [comment]
Indeed, I am using WSL, and I am running Docker. I'm not sure if this is the cause of the issue. Currently, I don't have other devices to test with. I also installed Git, but everything is on default settings—the same goes for WSL, which only has Docker installed.

**Comment by wdkflag** [comment]
However, I would like to ask another question, and I hope you do not mind. I want to know how this project performs on a Mac. I have a MacBook with an M3 Pro chip, and previously, I tried running it using Applio, which did not work. I also attempted to run the original RVC, but that did not work either. There is another software called Replay, which can run but is extremely slow. I would like to know the speed of this project on a Mac. Thank you.

**Comment by wdkflag** [comment]
@rsxdalv hello

**Comment by rsxdalv** [comment]
> @rsxdalv hello

Happy new year



> However, I would like to ask another question, and I hope you do not mind. I want to know how this project performs on a Mac. I have a MacBook with an M3 Pro chip, and previously, I tried running it using Applio, which did not work. I also attempted to run the original RVC, but that did not work either. There is another software called Replay, which can run but is extremely slow. I would like to know the speed of this project on a Mac. Thank you.

Some models support Mac, some do not. I think Bark, Musicgen Mac (Installable addon) supports it as well as some others, the RVC I have integrated might not work, but I think there are some Mac-compatible RVC forks.

> Indeed, I am using WSL, and I am running Docker. I'm not sure if this is the cause of the issue. Currently, I don't have other devices to test with. I also installed Git, but everything is on default settings—the same goes for WSL, which only has Docker installed.

I see, but this project was installed natively within Windows, not within WSL, right? On another note, you can run it using Docker. (And WSL has some benefits since Linux (Ubuntu + CUDA) has the best support, then Windows, and lastly Mac)
